docs-v2/integrations/all/github.mdx

```diff
@@ -57,5 +156,10 @@ _No setup guide yet._
 -   There are certain API methods that only work with an OAuth App that will not work with an App. Please check the Github documentation and look for a "Works with Github Apps" header under the endpoint.
 -   The `github-app-oauth` is a combination of a Github App and OAuth and should be used when a Github App would need approval to be installed into an organization.
 -   Nango supports initiating a connection with a GitHub App using the frontend SDK, but not directly from the [GitHub Marketplace](https://github.com/marketplace). Therefore, you should encourage users to install your GitHub App from your product, rather than from the GitHub Marketplace directly. This is a limitation we plan to fix in the future.
+-   GitHub's OAuth implementation supports the standard authorization code grant type but does not support the implicit grant type.
```

Comment:
This and none of the below are useful or informative gotchas. A lot of Nango's value prop is that these details are handled by Nango so our users don't have to worry about or even know these "gotchas". Please remove and note for other pull requests.